html/bleaching.py

```diff
@@ -0,0 +1,164 @@
+import bleach
+
+
+class Cleaner(bleach.Cleaner):  # pylint: disable=too-few-public-methods
```

Comment:
Does this need to subclass `bleach.Cleaner` ? 

Could this just be a `def get_cleaner(strict=True):` factory method that just returns `bleach.Cleaner(tags=tags, attributes=attributes, styles=styles, strip=True)` ?

Reply:
Sure, what you're suggesting is better because we're not overriding methods or functionalities here.